substrate/frame/support/src/storage/unhashed.rs

```diff
@@ -170,7 +170,7 @@ pub fn get_raw(key: &[u8]) -> Option<Vec<u8>> {
 }
 
 /// Put a raw byte slice into storage.
-///
+// TODO: this comment seems outdated
```

Comment:
True, it was removed several years ago while moving the runtime upgrade logic to `frame_executive`, so the comment should be removed as well.